sparse-11/etc/gbinder.conf

```diff
@@ -0,0 +1,2 @@
+[General]
+ApiLevel = 29
```

Comment:
30?

Reply:
No support for api level 30 in gbinder yet.

Reply:
It seems we can set 30 here even before the support is merged.